deploy/stacks/pipeline.py

```diff
@@ -637,14 +637,15 @@ def set_backend_stage(self, target_env, repository_name):
                 prod_sizing=target_env.get('prod_sizing', True),
                 quicksight_enabled=target_env.get('enable_quicksight_monitoring', False),
                 enable_cw_rum=target_env.get('enable_cw_rum', False),
-                enable_cw_canaries=target_env.get('enable_cw_canaries', False),
+                enable_cw_canaries=target_env.get('enable_cw_canaries', False) and target_env.get("custom_auth", None) == None,
```

Comment:
I don't understand this line. Why do we need to add this guardrail?

Reply:
cw_canaries create a cognito user in order to work. Thus, putting this guardrail to make sure that cw_canaries is not created when the custom auth is set in which cognito stack is not present

Reply:
Ahh thanks!